pkg/datapath/iptables/iptables.go

```diff
@@ -590,40 +626,44 @@ func (m *IptablesManager) doCopyProxyRules(prog iptablesInterface, table string,
 				}).WithError(err).Warn("Unable to parse TPROXY rule, disruption to traffic selected by L7 policy possible")
 				continue
 			}
+
 			copyRule := append([]string{"-t", table}, args...)
-			log.WithField(logfields.Object, logfields.Repr(copyRule)).Debugf("Copying %s TPROXY rule from %s to %s", prog, oldChain, newChain)
```

Comment:
After adding the debug logs in `runProg()`/`runProgCombinedOutput()` I thought it would be redundant to keep these logs.

Sure, we'll see an error that only mentions the actual iptables command that failed rather than `Copying [..] TPROXY rule from $OLD_CHAIN to $CHAIN`, but that's probably not an issue as I think we can get already all the context we need just from the plain iptables command that failed (also, if we end up wrapping the error with the additional TPROXY context, why shouldn't we do that with every other rule?)

Reply:
> After adding the debug logs in `runProg()`/`runProgCombinedOutput()` I thought it would be redundant to keep these logs.

I agree. But when I was reviewing this commit, I didn't know about the last commit which actually has the related changes :wink: :wink: :wink: